package.json

```diff
@@ -154,7 +154,8 @@
     "url-loader": "3.0.0",
     "webpack": "4.41.2",
     "webpack-cli": "3.3.10",
-    "webpack-livereload-plugin": "2.2.0"
+    "webpack-livereload-plugin": "2.2.0",
+    "yaml-jest": "^1.2.0"
```

Comment:
Nit :  Are we planning to use directly in any test files as JSON? Please clarify.
Also, Could you please update in the description about this change too, so that context would be clear when referred later?

Reply:
It's a dev-dependency required by jest (unit test runner) to be able to import the testids.yaml file. `yaml-jest` is a transformer that takes a yaml file and transforms it into a js module consumable by jest.